samples/notebooks/powershell/Samples/Using-Microsoft-DataFrame.ipynb

```diff
@@ -0,0 +1,260 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "91ee47dc",
+   "metadata": {
+    "polyglot_notebook": {
+     "kernelName": "pwsh"
+    }
+   },
+   "source": [
+    "# Overview\n",
+    "\n",
+    "This is an example of using [DataFrame](https://github.com/nightroman/DataFrame) module in .NET Ineractive notebooks using PowerShell"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4359abf4",
+   "metadata": {},
+   "source": [
+    "First we need to load the correct version of the `Microsoft.Data.Analysis` package, along with `Microsoft.DotNet.Interactive.ExtensionLab` which performs formatting"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2e425e3a",
+   "metadata": {
+    "polyglot_notebook": {
+     "kernelName": "csharp"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "#r \"nuget: Microsoft.Data.Analysis, 0.22.2\"\n",
+    "#r \"nuget: Microsoft.DotNet.Interactive.ExtensionLab, 1.0.0-beta.25177.1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fece74ec",
+   "metadata": {},
+   "source": [
+    "Now we install the module"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "befbe206",
+   "metadata": {
+    "polyglot_notebook": {
+     "kernelName": "pwsh"
+    }
+   },
+   "outputs": [],
```

Comment:
Feel free to save this with outputs after running it.

Reply:
well, feels like GitHub still does not display them correctly :(

Reply:
Here's an example:

https://github.com/dotnet/interactive/blob/main/samples/connect-wpf/readme.ipynb